### DISCUSSION TITLE
PTZ NETIP protocol

### DISCUSSION THREAD

**Discussion by bjct42** [ZoneMinder/zoneminder#3981] [General]
Hello, do you plan to integrate the NETIP protocol in the management of PTZ?

**Comment by connortechnology** [comment]
I have no plans to.  If someone were to sponsor the effort I'm sure it could be done fairly quickly.  Or someone else might.

**Comment by bjct42** [comment]
helas, I’m not a developper, otherwise I would have gladly done the task. Thank you for the answer.
But I found this, if it can help motivated people.
https://pypi.org/project/python-dvr/
